src/ts/components/labels/MetadataLabel.ts

```diff
@@ -34,6 +38,7 @@ export interface MetadataLabelConfig extends LabelConfig {
  * @category Labels
  */
 export class MetadataLabel extends Label<MetadataLabelConfig> {
+  private adMessage: string = '';
```

Comment:
This component should be generic and therefore shouldn't have any ad or non-ad related instance properties. It it possible to get rid of this?